### ISSUE TITLE
Failed to inject frame into filter network: Invalid argument

### ISSUE THREAD

**Issue by michael5wong** [mluggy/teslabox#33]
The event.json is creating (and storing) the gif, pinging telegram, but looks like is failing on creating the full video (error below).  Any ideas?

> [queue/archive] /mnt/usb/TeslaCam/SentryClips/2023-09-29_12-09-30/event.json failed: Error: Command failed: ffmpeg -y -hide_banner -loglevel error -i /root/teslabox/src/assets/favicon.ico -ss 42 -t 5 -i /mnt/ram/3011c6ffb670ba4495e5483c4baef524083e58a9.mp4 -ss 47 -t 3 -i /mnt/ram/c7c03c3330e596ce0c558e7183d6b8234f293c15.mp4 -ss 50 -t 8 -i /mnt/ram/d1ad7f33ad1577cc7de033943d3f49ea3f026326.mp4 -filter_complex "[0]scale=25:25 [icon]; [1][2][3]concat=n=3:v=1 [all]; [all]drawtext=fontfile='/root/teslabox/src/assets/FreeSans.ttf':fontcolor=white:fontsize=25:borderw=1:bordercolor=black@1.0:x=38:y=930:text='TeslaBox wongmobile Sentry %{pts\:localtime\:1696003735}' [video]; [video][icon]overlay=8:928" -preset veryfast -crf 19 -r 24 /mnt/ram/c53291505dc8597e37944707cf748633ee4c7355.mp4 [Parsed_concat_1 @ 0x556a298d10] Input link in0:v0 parameters (size 1448x938, SAR 1:1) do not match the corresponding output link in0:v0 parameters (2896x1876, SAR 1:1) [Parsed_concat_1 @ 0x556a298d10] Failed to configure output pad on Parsed_concat_1 Error reinitializing filters! Failed to inject frame into filter network: Invalid argument Error while processing the decoded data for stream #3:0

**Comment by aphan789** [comment]
Getting this error too, did you find a fix?

```
queue/archive] /mnt/usb/TeslaCam/SentryClips/2023-09-30_15-55-47/event.json failed: Error: Command failed: ffmpeg -y -hide_banner -loglevel error -i /root/teslabox/src/assets/favicon.ico -ss 22 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -ss 24 -t 10 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 34 -t 5 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 39 -t 8 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 47 -t 6 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 53 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -filter_complex "[0]scale=25:25 [icon]; [1][2][3][4][5][6]concat=n=6:v=1 [all]; [all]drawtext=fontfile='/root/teslabox/src/assets/FreeSans.ttf':fontcolor=white:fontsize=25:borderw=1:bordercolor=black@1.0:x=38:y=930:text='TeslaBox Teslabo Sentry %{pts\:localtime\:1696103669}' [video]; [video][icon]overlay=8:928" -preset veryfast -crf 23 -r 24 /mnt/ram/5fa069849192617b76d691335349b166aefda0e7.mp4 [Parsed_concat_1 @ 0x5598ba67e0] Input link in0:v0 parameters (size 1448x938, SAR 1:1) do not match the corresponding output link in0:v0 parameters (2896x1876, SAR 1:1) [Parsed_concat_1 @ 0x5598ba67e0] Failed to configure output pad on Parsed_concat_1 Error reinitializing filters! Failed to inject frame into filter network: Invalid argument Error while processing the decoded data for stream #6:0
```

**Comment by mluggy** [comment]
what duration do you have for sentry clips? can you change to 30 seconds to see if it sorts it out?On 1 Oct 2023, at 0:32, aphan789 ***@***.***> wrote:﻿
Getting this error too, did you find a fix?
queue/archive] /mnt/usb/TeslaCam/SentryClips/2023-09-30_15-55-47/event.json failed: Error: Command failed: ffmpeg -y -hide_banner -loglevel error -i /root/teslabox/src/assets/favicon.ico -ss 22 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -ss 24 -t 10 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 34 -t 5 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 39 -t 8 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 47 -t 6 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 53 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -filter_complex "[0]scale=25:25 [icon]; [1][2][3][4][5][6]concat=n=6:v=1 [all]; ***@***.***:x=38:y=930:text='TeslaBox Teslabo Sentry %{pts\:localtime\:1696103669}' [video]; [video][icon]overlay=8:928" -preset veryfast -crf 23 -r 24 /mnt/ram/5fa069849192617b76d691335349b166aefda0e7.mp4 [Parsed_concat_1 @ 0x5598ba67e0] Input link in0:v0 parameters (size 1448x938, SAR 1:1) do not match the corresponding output link in0:v0 parameters (2896x1876, SAR 1:1) [Parsed_concat_1 @ 0x5598ba67e0] Failed to configure output pad on Parsed_concat_1 Error reinitializing filters! Failed to inject frame into filter network: Invalid argument Error while processing the decoded data for stream #6:0

**Comment by aphan789** [comment]
I have it set high and 30seconds already, should I lower the quality?

**Comment by michael5wong** [comment]
> what duration do you have for sentry clips? can you change to 30 seconds to see if it sorts it out?On 1 Oct 2023, at 0:32, aphan789 ***@***.***> wrote:﻿
> 
> Getting this error too, did you find a fix?
> 
> queue/archive] /mnt/usb/TeslaCam/SentryClips/2023-09-30_15-55-47/event.json failed: Error: Command failed: ffmpeg -y -hide_banner -loglevel error -i /root/teslabox/src/assets/favicon.ico -ss 22 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -ss 24 -t 10 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 34 -t 5 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 39 -t 8 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 47 -t 6 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 53 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -filter_complex "[0]scale=25:25 [icon]; [1][2][3][4][5][6]concat=n=6:v=1 [all]; ***@***.***:x=38:y=930:text='TeslaBox Teslabo Sentry %{pts\:localtime\:1696103669}' [video]; [video][icon]overlay=8:928" -preset veryfast -crf 23 -r 24 /mnt/ram/5fa069849192617b76d691335349b166aefda0e7.mp4 [Parsed_concat_1 @ 0x5598ba67e0] Input link in0:v0 parameters (size 1448x938, SAR 1:1) do not match the corresponding output link in0:v0 parameters (2896x1876, SAR 1:1) [Parsed_concat_1 @ 0x5598ba67e0] Failed to configure output pad on Parsed_concat_1 Error reinitializing filters! Failed to inject frame into filter network: Invalid argument Error while processing the decoded data for stream #6:0
> 
> 
> 
> 
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

I'm on 30s, highest. I'll lower the quality to medium and experiment?  But never adjusted higher than 30s.

**Comment by michael5wong** [comment]
> > what duration do you have for sentry clips? can you change to 30 seconds to see if it sorts it out?On 1 Oct 2023, at 0:32, aphan789 ***@***.***> wrote:﻿
> > 
> > Getting this error too, did you find a fix?
> > 
> > queue/archive] /mnt/usb/TeslaCam/SentryClips/2023-09-30_15-55-47/event.json failed: Error: Command failed: ffmpeg -y -hide_banner -loglevel error -i /root/teslabox/src/assets/favicon.ico -ss 22 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -ss 24 -t 10 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 34 -t 5 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 39 -t 8 -i /mnt/ram/5b33657651c76c0aa3df4221d5a438ca96adb71c.mp4 -ss 47 -t 6 -i /mnt/ram/2f05b96ea84624a2fe74fb2ff25fc1ece2f45696.mp4 -ss 53 -t 2 -i /mnt/ram/a379eb5c864b3603cdeafcdc489b3a1f40bb44ff.mp4 -filter_complex "[0]scale=25:25 [icon]; [1][2][3][4][5][6]concat=n=6:v=1 [all]; ***@***.***:x=38:y=930:text='TeslaBox Teslabo Sentry %{pts\:localtime\:1696103669}' [video]; [video][icon]overlay=8:928" -preset veryfast -crf 23 -r 24 /mnt/ram/5fa069849192617b76d691335349b166aefda0e7.mp4 [Parsed_concat_1 @ 0x5598ba67e0] Input link in0:v0 parameters (size 1448x938, SAR 1:1) do not match the corresponding output link in0:v0 parameters (2896x1876, SAR 1:1) [Parsed_concat_1 @ 0x5598ba67e0] Failed to configure output pad on Parsed_concat_1 Error reinitializing filters! Failed to inject frame into filter network: Invalid argument Error while processing the decoded data for stream #6:0
> > 
> > 
> > 
> > 
> > 
> > —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
> 
> I'm on 30s, highest. I'll lower the quality to medium and experiment?  But never adjusted higher than 30s. 

And same error on 30s, medium quality.

**Comment by mluggy** [comment]
what cameras are you running? could it be they changed the resolution? can you send the raw files by telegram (@mluggy)?

**Comment by aphan789** [comment]
Sure, fyi it's able to upload the clips to s3. I'm on HW4

**Comment by mluggy** [comment]
i think we're getting there. I haven't seen HW4 files yet, maybe something has changed. if you can compress the entire folder for that Sentry Event and send via telegram (mluggy) or mail (same, with gmail.com)

**Comment by aphan789** [comment]
Just sent it over on telegram

**Comment by mluggy** [comment]
ok we seem to be able to narrow it down to 2 issues:
1- telegram chat has to be "initialized", just start a conversation with the bot you've created and hit "hi"
2- cinematic mode (for sentry) doesn't seem to work because of newer/higher resolutions. I'm getting those measurments from @aphan789 and will send a fix

**Comment by mluggy** [comment]
cinematic (and additional higher resolution/cropping/scaling) were fixed on 1.0.0, which will determine the hwVersion configuration upon box initialization (either 3 or 4).

thanks @aphan789 for taking the time to test this with me.

let me know if you're still experiencing issues.
